.github/workflows/tag-and-release.yml

```diff
@@ -20,7 +20,9 @@ jobs:
       - name: Configure repository 🛠
         run: |
           git remote rename origin stackhpc &&
-          git remote add origin "https://github.com/openstack/$(basename $(pwd)).git"
+          git remote add origin "https://github.com/openstack/$(basename $(pwd)).git" &&
+          git fetch origin &&
+          git push --follow-tags
```

Comment:
I think we need to be specific about which remote we are pushing to.

Reply:
Yeah makes sense to be clear with regards to where the push is targeting. This has been addressed in the latest commit.